README.md

```diff
@@ -60,12 +60,17 @@ As with the desktop fonts, they are made available both as variable and static f
 ## Editors
 
 ### Visual Studio Code
+Set the font family
+
+```json
+    "editor.fontFamily": "'Monaspace Neon Var', monospace",
```

Comment:
Actually, AFAIK VS Code doesn't support variable faces so well right now, it's better to reference the static builds

```suggestion
    "editor.fontFamily": "'Monaspace Neon', monospace",
```